projectdoc/infrastructure.tex

```diff
@@ -0,0 +1,24 @@
+
+\section{Infrastruktur}
+\label{sec:Infrastruktur}
+
+\subsection{Continuous Integration}
+\label{sub:Continuous Integration}
+
+In unserem Projekt wird \ac{CI} einerseits für das automatische Erstellen dieser Dokumentation mi LaTeX verwendet, und andererseits, um automatische Tests für die Backend-Implementation auszuführen. Beides wird im Folgenden kurz beschrieben.
+
+\subsubsection{Erstellen der LaTeX-Dokumentation}
+\label{subsub:Erstellen der LaTeX-Dokumentation}
+
+Bei jedem Git Commit in das Repository der Dokumentation \cite{github:PlazaRoute-doc} wird mit Continuous Integration ein PDF erstellt. So wird sichergestellt, dass die Dokumentation keine syntaktischen Fehler enthält. Das PDF wird in unser internes JIRA hochgeladen und direkt dem Task angehängt, der zum Commit gehört.
```

Comment:
ich frage mich, wie wir generell mit Abkürzungen umgehen, verwenden wir in jedem Fall immer \ac{...}

Reply:
Ja, ausser es soll wirklich nur das Kürzel verwendet werden...
- `\ac` ergibt bei erster Verwendung die ausgeschriebene Variante, nacher immer das Akronym
- `\acs` gibt immer das Acronym

Also grundsätzlich verwende ich immer `\ac`, ausser die ausgeschriebene Variante passt dort nicht in den Satz

Reply:
Ich verwende immer `\ac`, d.h bei erster Verwendung im Dokument wird die Abkürzung ausgeschrieben. Ausser es passt dort gerade nicht in den Satz, dann kann man die Kurzschreibweise mit `\acs` forcieren

Reply:
ja weil du im Abschnitt mehrmals Continuous Integration verwendest, wunderte ich mich, ob es bewusst ist

Reply:
habe das bewusst ausgeschrieben zur Lesbarkeit, aber es gibt kein `\ac` Befehl um explizit die ausgeschriebene Version zu verwenden